.travis.yml

```diff
@@ -24,6 +30,28 @@ before_install:
   - mkdir -p ${HOME}/.m2
   - cp settings.xml $HOME/.m2
 
+  - export PREVIEW_NAMESPACE=$(echo "v$TRAVIS_BRANCH-$TRAVIS_BUILD_NUMBER"|  tr '[:upper:]' '[:lower:]'|tr . -)
+  - ORG='activiti'
+  - export HELM_RELEASE_NAME="$TRAVIS_BRANCH-$TRAVIS_BUILD_NUMBER"
+  - export GLOBAL_GATEWAY_DOMAIN="35.242.205.159.nip.io"
+  - export GATEWAY_HOST="gateway.$PREVIEW_NAMESPACE.$GLOBAL_GATEWAY_DOMAIN"
+  - export SSO_HOST="identity.$PREVIEW_NAMESPACE.$GLOBAL_GATEWAY_DOMAIN"
+  - curl -Ls -o updatebot.jar   https://repo1.maven.org/maven2/io/jenkins/updatebot/updatebot/1.1.53/updatebot-1.1.53.jar
```

Comment:
extract version in a UPDATEBOT_VERSION variable

Reply:
also JX_RELEASE_VERSION=1.0.24

Reply:
done